cmd/renterd/main.go

```diff
@@ -148,6 +148,37 @@ func main() {
 		node.AutopilotConfig
 	}
 	autopilotCfg.RevisionSubmissionBuffer = api.BlocksPerDay
+
+	// TODO: the following flags will be deprecated in v1.0.0 in favor of
+	// environment variables to ensure we do not ask the user to pass sensitive
+	// information via CLI parameters.
+	flag.StringVar(&dbCfg.password, "db.password", "", "password for the database to use for the bus - can be overwritten using RENTERD_DB_PASSWORD environment variable")
+	flag.StringVar(&busCfg.apiPassword, "bus.apiPassword", "", "API password for remote bus service - can be overwritten using RENTERD_BUS_API_PASSWORD environment variable")
```

Comment:
I'd add "deprecated" to the descriptions as well. So you don't need to use the flag to know that they are deprecated

Reply:
👍 good idea